.circleci/config.yml

```diff
@@ -1,41 +1,12 @@
 version: 2.1
 
 orbs:
-  plugin-ci: mattermost/plugin-ci@volatile
+  plugin-ci: mattermost/plugin-ci@0.1.0
```

Comment:
This is the only file that should be synced with https://github.com/mattermost/mattermost-plugin-demo/blob/master/.circleci/config.yml instead of https://github.com/mattermost/mattermost-plugin-starter-template. The version of the demo plugin contains certain pieces specifically for Mattermost owned repos.